base/config.js

```diff
@@ -81,17 +89,17 @@ module.exports.schema = {
 }
 
 module.exports.mergeDefaults = (opts, schema) => {
-  if (!opts || !schema) throw new Error('schema.mergeDefaults(opts, schema): opts and schema objects are required')
+  if (!opts || !schema) throw new Error('opts and schema objects are required')
   return reduce(keys(schema), (accum, key) => {
-    accum[key] = opts[key] !== undefined ? opts[key] : schema[key].defaultValue()
+    accum[key] = opts[key] !== undefined ? opts[key] : schema[key].defaultValue(opts[key], opts)
```

Comment:
I don't understand the context to this change (although that may just be my rusty JS) - is this part of the session tracking implementation?

Reply:
The `defaultValue` function previously operated on each schema item entirely independently. Now the default value for `autoCaptureSessions` is dependent on the original config object, so `opts` (the user-provided configuration object) needs to get passed in to the `defaultValue` call.